PrivateCollections/Ethiopia/MentasnotAbbuhay/MEAG002.xml

```diff
@@ -0,0 +1,404 @@
+                                  <explicit xml:lang="gez">አርእየኒ፡ገጸከ፡ ኃይለከ፡ በግርማከ፡ መስቀል፡ መዋዴ፡ ፀር፡ መስቀል፡ መግረሬ፡ ፀር፡ መስቀል፡ ነሳቴ፡ ፀር፡ መስቀለ፡ ኢየሱስ ክርስቶስ፡
+                                      አግርሮሙ፡ ለመናፍስት፡ ርኩሳን፡ አዶናይ፡ ቅዱስ፡ እግዚአብሔር፡ ኃያል፡ አድኅኖ፡ ለገብርከ። </explicit> 
+                              </msItem>
+                              <msItem xml:id="ms_i2.4">
+                                  <locus from="81" to="101"/> 
+                                  <title type="complete">Prayer for Thursday</title>
+                                  <textLang mainLang="gez"/>
+                                  <incipit xml:lang="gez">ዘሐሙስ፡ በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ። ሸራህያ፡ አልሻዳይ፡ እልመክኦን፡ አልፋ፡ ወዖ፡ ኢየሱስ፡ ክርስቶስ፡ ወልደ፡ 
+                                      እግዚአብሔር፡ ሕያው፡ ወወልደ፡ ማርያም፡ ሥግው፡ ዘበመስቀልከ፡ አፅራዕከ፡ግብሮ፡ ለዲያብሎስ ። ከማሁ፡ አፅርዕ፡ ግብረ፡ </incipit>
+                                  <explicit xml:lang="gez">ወወረደ፡ መንፈስ፡ ቅዱስ፡ ላዕለ፡ ዮርዳኖስ። ከማሁ፡ ሥጥጥ፡ ጽልመተ፡ ሕሊናሁ፡ወአቅልል፡ ማዕሠረ፡ አባላቲሁ፡ ዮፍታሔ፡ ዮፍታሔ፡
+                                      ዮፍታሔ፡ ፍታሕ፡ ሥራዮሙ፡ ለመሠርያን፡ እምላዕለ፡ ገብርከ። </explicit>
+                              </msItem>
+                              <msItem xml:id="ms_i2.5">
+                                  <locus from="102" to="125"/> 
+                                  <title type="complete">Prayer for Friday</title>
+                                  <textLang mainLang="gez"/>
+                                  <incipit xml:lang="gez">ዘዓርብ፡ በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ። ራኮን፡ ራፎን፡ ፍታሕ፡ ጕሕኤል፡ ፍታሕ፡ አማኑኤል፡ ፍታሕ፡ ወራፎን፡ ፍታሕ፡ 
+                                      ጕልታ፡ ፍታሕ፡ ወዘርዝር፡ በከመ፡ ፈታሕኮሙ፡ ለጴጥሮስ፡ ወለጳውሎስ፡ ከማሁ፡ ፍትሐኒ፡ ለገብርከ። ዘፈታሕኮሙ፡ ለጴጥሮስ፡ አርታ፡ ፍጥረታተ፡ </incipit>
+                                  <explicit xml:lang="gez"> ርቆ፡ አድኀኖ፡ እምሕማመ፡ ብድብድ፡ ለገብርከ። አልመድክክን፡ አዚዚ፡ አበረቤ፡ ባኦስ፡ ኢየሱስ፡ ክርስቶስ፡ አድኅኖ፡ እምባርያ፡ 
+                                      ዘየሐንቅ፡ ክሳደ፡ ወያበኀርር፡ በሕልም፡ አድኅኖ፡ በላዕለ፡ ገብርከ። ለዓለመ፡ ዓለም፡ አሜን። </explicit>
+                              </msItem>
+                              <msItem xml:id="ms_i2.6">
+                                  <locus from="126" to="152"/> 
+                                  <title type="complete">Prayer for Saturday</title>
+                                  <textLang mainLang="gez"/>
+                                  <incipit xml:lang="gez">ዘቀደም፡ በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ። ጸሎት፡ በእንተ፡ ድንጋጼ፡ አጋንንት፡ በእንተ፡ ገበዋቶን። ወእምብዝሐ፡
+                                      ድንጋፄ፡ ይቤ፡ ሰሎሞን፡ ምንት ይመስል፡ ፍጥረቶሙ፡ ለነሀብት፡ ርኩሳን፡ ወድዩአን፡ በስመ፡ እግዚአብሔር፡ አሸፒራ፡ ወላአሺ፡ </incipit>
+                                  <explicit xml:lang="gez">ጸላዊ፡ ዘትሰርቅ፡ በዓይነ፡ ጠባይዕ፡ ነፋሳዊ ኢትሥግር፡ በሥራይ። ሑት፡ ዘትሠርቅ፡ በዓይነ፡ ጠባይዕ፡ ማያዊ፡ ኢትሥግር፡ በሥራይ። 
+                                      በእሉ፡ አስማት፡ ከዋክብት፡ አድኅኖ፡ እምሕማመ፡ ሥራይ፡ ለገብርከ። </explicit>
+                              </msItem>
+                              <msItem xml:id="ms_i2.7">
+                                  <locus from="153" to="173"/> 
+                                  <title type="complete" >Prayer for Sunday</title>
```

Comment:
Please remove the space after "complete". Unreasonable space in xml-strings may cause trouble to the app.